cyhy_report/customer/report.mustache

```diff
@@ -590,12 +592,12 @@ As you review the report, you may have additional questions. Check out the answe
         {\RCSourceSansProTwelve <<#calc.address_count_up_flag>>Increase of \textbf{\numprint{<<&calc.address_count_up>>}}<</calc.address_count_up_flag>>
         <<#calc.address_count_down_flag>>Decrease of \textbf{\numprint{<<&calc.address_count_down>>}}<</calc.address_count_down_flag>>
         <<#calc.address_count_flat_flag>>No Change<</calc.address_count_flat_flag>>}
-      \end{minipage}  % END: ADDRESSES OWNED
+      \end{minipage}  % END: ASSETS OWNED
       \begin{minipage}[t]{0.1\textwidth}
          \  % Spacer column
       \end{minipage}
       \begin{minipage}[t]{0.45\textwidth}
-        \textcolor{rc-dark-green}{\RCSourceSansProBoldFourteen ADDRESSES SCANNED}
+        \textcolor{rc-dark-green}{\RCSourceSansProBoldFourteen LAST SCOPE UPDATE}
```

Comment:
@jeffkause Per our discussion, since the "LAST SCOPE UPDATE" change is going to happen in a future PR, we just want to do this for now, right?
```suggestion
        \textcolor{rc-dark-green}{\RCSourceSansProBoldFourteen ASSETS SCANNED}
```

Reply:
Yes, since we are putting the change request for the Scope Last Update in a separate request, I changed it back tot eh original wording.

Reply:
I see you did that in https://github.com/cisagov/cyhy-reports/pull/91/commits/a07b6e93e298c2c536a8986ca5b3f9460f82a282.  Thanks!